sklego/meta/outlier_classifier.py

```diff
@@ -29,8 +30,16 @@ def fit(self, X, y=None):
         X, y = check_X_y(X, y, estimator=self)
         if not self._is_outlier_model():
             raise ValueError("Passed model does not detect outliers!")
+        if not hasattr(self.model, 'decision_function'):
+            raise ValueError('Passed model does not have a `decision_function` method. This is required for '
```

Comment:
This is a nitpick, but it would be better to also list the model in question. Maybe instead raise something like: 

```python
raise ValueError(f'Passed model {self.model} does not have a `decision_function` '
                 f'method. This is required for `predict_proba` estimation.')
```